pkg/services/ngalert/models.go

```diff
@@ -15,6 +14,9 @@ type AlertDefinition struct {
 	Name      string
 	Condition string
 	Data      []eval.AlertQuery
+	Updated   int64
```

Comment:
So it seems this can be time.Time (if you keep it). Did some xorm digging/debug tracing. If the db type is a BIGINT, and the go type is time.Time, xorm calls `t.Unix()`.